tests/errors.test.ts

```diff
@@ -12,70 +12,40 @@ fetchMock.enableMocks();
 
 jest.setTimeout(100 * 1000);
 
+// @TODO: Have to review this in more detail
 describe('Test on updates', () => {
   beforeEach(() => {
     fetchMock.mockReset();
   });
 
-  test(`Throw MeiliSearchCommunicationError when throwned error is not MeiliSearchApiError`, async () => {
+  test(`Throw MeilisearchRequestError when throwned error is not MeiliSearchApiError`, async () => {
     fetchMock.mockReject(new Error('fake error message'));
     const client = new MeiliSearch({ host: 'http://localhost:9345' });
     try {
       await client.health();
     } catch (e: any) {
-      expect(e.name).toEqual('MeiliSearchCommunicationError');
+      expect(e.name).toEqual('MeiliSearchRequestError');
     }
   });
 
-  test(`Not throw MeiliSearchCommunicationError when throwned error is MeiliSearchApiError`, async () => {
-    fetchMock.mockReject(
-      new MeiliSearchApiError(
-        {
-          message: 'Some error',
-          code: 'some_error',
-          type: 'random_error',
-          link: 'a link',
-        },
-        404,
-      ),
-    );
-
-    const client = new MeiliSearch({ host: 'http://localhost:9345' });
-    try {
-      await client.health();
-    } catch (e: any) {
-      expect(e.name).toEqual('MeiliSearchApiError');
-    }
-  });
-
-  test('MeiliSearchApiError can be compared with the instanceof operator', async () => {
-    fetchMock.mockReject(
-      new MeiliSearchApiError(
-        {
-          message: 'Some error',
-          code: 'some_error',
-          type: 'random_error',
-          link: 'a link',
-        },
-        404,
-      ),
-    );
-
-    const client = new MeiliSearch({ host: 'http://localhost:9345' });
-    try {
-      await client.health();
-    } catch (e: any) {
-      expect(e instanceof MeiliSearchApiError).toEqual(true);
-    }
```

Comment:
This test case was removed, because it's no longer applicable, `MeiliSearchApiError` doesn't get caught anymore to be re-thrown unnecessarily.